### ISSUE TITLE
Body and pubic hair also removed.

### ISSUE THREAD

**Issue by VamDazzler** [acidbubbles/vam-improved-pov#10]
Working as coded, obviously, but intuitively unexpected.
Some thought might be given to how to let the user whitelist which hairs should remain.

**Comment by acidbubbles** [comment]
Yeah I was aware of this when I implemented the new hair system. A while (or black) list could be an option, or maybe simply using tags they might be automatically discoverable.

**Comment by acidbubbles** [comment]
I can probably use the Face/Head tags to decide whether to hide or not the hair (this and tagless)
